overlord/devicestate/export_test.go

```diff
@@ -537,7 +539,7 @@ func MockEncryptionSetupDataInCache(st *state.State, label string) (restore func
 			EncryptedDevice: "/dev/mapper/ubuntu-data",
 		},
 	}
-	esd = install.BuildEncryptionSetupData(labelToEncData)
+	esd = install.MockEncryptionSetupData(labelToEncData)
 	st.Cache(encryptionSetupDataKey{label}, esd)
 	return func() { CleanUpEncryptionSetupDataInCache(st, label) }
```

Comment:
I missed how CleanUp... was used, is it used somewhere in the tests or just here? if just here it probably shouldn't be exported

Reply:
Also in the tests: https://github.com/snapcore/snapd/blob/master/overlord/devicestate/devicestate_install_api_test.go#L391